### ISSUE TITLE
Dead link in the documentation

### ISSUE THREAD

**Issue by kuba-orlik** [microsoft/TypeScript-Website#2983]
The "DOM type definitions" link at https://typescript-v2-784.vercel.app/docs/handbook/dom-manipulation.html leads to a 404 page

**Comment by hwahyeon** [comment]
It works now (Checked in Feb 2024) 👍

**Comment by jakebailey** [comment]
I don't think it does. The link is to https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts (a file I removed!)

**Comment by kuba-orlik** [comment]
Yep, confirming clicking "DOM type definitions" in https://typescript-v2-784.vercel.app/docs/handbook/dom-manipulation.html still gives me 404
